app-text/pad/licenses/Pad

```diff
@@ -0,0 +1,64 @@
+
+Usage
+----------
+Run the program. Type stuff. Press the buttons. It's not rocket-science :-)
+
+NOTE: Information such as the length of each page is stored in attributes. So if the attributes get trashed somehow (rmattr, moving to a non-Be filesystem, etc.), Pad will no longer open the files. However, you should still be able to open the file in a text editor such as StyledEdit. The pages will all be run together, but the text will all be there.
+
+Multiple pads
+--------------------
+Pad can only have one file open at a time; however, you can have multiple instances of Pad open at once, each with a different file.  To create multiple pads, just duplicate the file "Pad_default" in /boot/system/settings, and rename/move it wherever you want. Just double-click this file to open it in Pad. *
+* At this time, double-clicking does not work. I believe this to be a problem with AAPR, but I'm still looking into it. However, you can still open non-default pad files in two ways:
+1) from the command line: "Pad my_pad &" where 'my_pad' is the name of your pad file.
+2) by dragging the pad file onto the Pad icon.
+
+Future improvements
+---------------------------------
+• Double-click to open pad files. This is my top priority.
+
+• If you resize the window quickly, you'll see a "rubber-band" effect on the window elements. This should be gone within a week or so, although a solution I'm happy with will probably take a little longer. Also, I plan to make the interface look a bit better -- I consider it quite ugly right now.
+
+• Multiple pads open with a single instance of the app.
+
+• Remember window location/size.
+
+• Allow user to configure default pad location. This will probably happen as soon as one of the several preferences managers becomes a de facto standard.
+
+• Allow import of text files.
+
+• More editing commands, e.g. Select All, Clear, possibly Undo.
+
+• Clicking on page number brings up dialog box that allows jumping to any page.
+
+Known Bugs
+-------------------
+None that I know of. I'm sure I'll hear about a whole bunch as soon as other people get their hands on it though.
+
+Legal stuff
+-----------------
+Pad is freeware. I certainly don't mind donations to my college fund, but I'll feel guilty if you give me more than $5 for this :-). You can distribute it freely, as long as 1) this file accompanies it, 2) neither file is modified in any way, and 3) you don't charge more than the cost of distribution. If you want to include this with a commercial product, please get my permission first.
+
+As always, I am not legally responsible for any damage caused directly or indirectly by this application. Use it at your own risk.
```

Comment:
Please omit all but the "Legal stuff" section. That's the only section we care about for the license.